src/clickonce/launcher/Program.cs

```diff
@@ -0,0 +1,139 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Win32;
+using System;
+using System.Deployment.Application;
+using System.IO;
+
+namespace Microsoft.Deployment.Launcher
+{
+    static class Program
+    {
+        /// <summary>
+        /// The main entry point for the application.
+        /// </summary>
+        [STAThread]
+        static void Main()
+        {
+            try
+            {
+                StartLoggingIfConfigured();
```

Comment:
Typically this would be kept out of the `try` since its state isn't obvious (e.g. how far was the `Logger` along in initialization before it threw an exception).

Reply:
Makes sense - will move it.